gke-to-gke-vpn/README.md

```diff
@@ -148,17 +144,11 @@ those clusters.
 1. Kubernetes Engine >= 1.10.0-gke.0
 
 ### Setup
-1. Pick unique names for PROJECT1 and PROJECT2 and set the variables.
```

Comment:
Add a step for setting or verifying the default project is the one they want to use